src/connection-manager/connections.ts

```diff
@@ -1,22 +1,24 @@
+/* eslint-disable max-lines-per-function  -- Disable for this file with complex websocket rules. */
 import WebSocket from 'ws'
+import { LedgerEntryResponse, TxResponse } from 'xrpl'
 
 import {
   query,
   saveNodeWsUrl,
   clearConnectionsDb,
   getNetworks,
 } from '../shared/database'
-import {
-  DatabaseValidator,
-  Fee,
-  StreamLedger,
-  StreamManifest,
-  ValidationRaw,
-} from '../shared/types'
+import { Fee, LedgerResponseCorrected } from '../shared/types'
 import logger from '../shared/utils/logger'
 
-import agreement from './agreement'
-import { handleManifest } from './manifests'
+import {
+  getAmendmentLedgerEntry,
+  handleWsMessageLedgerEnableAmendments,
+  handleWsMessageLedgerEntryAmendments,
+  handleWsMessageSubscribeTypes,
+  handleWsMessageTxEnableAmendments,
+  subscribe,
+} from './utils'
```

Comment:
IMO this file should be called something like `wsHandling` (probably a better name is needed) because it's storing all the websocket handling. `utils` is too vague.